src/GraphODataTemplateWriter/CodeHelpers/Android/TypeHelperAndroid.cs

```diff
@@ -37,6 +37,13 @@ public static string GetTypeString(this OdcmType @type)
                     return "com.microsoft.graph.model.DateOnly";
                 case "Binary":
                     return "byte[]";
+                case "Stream":
+                    return "java.io.InputStream";
+                case "Duration":
+                    return "javax.xml.datatype.Duration";
```

Comment:
@iambmelt 
Have you validated that this type succeeds the de-serialization? If this works, can you please extract this change and the TimeOfDay change and re-submit with appropriate unit tests?